kivy/uix/camera.py

```diff
@@ -37,7 +37,7 @@ class Camera(Image):
     '''Camera class. See module documentation for more information.
     '''
 
-    play = BooleanProperty(True)
+    play = BooleanProperty(False)
```

Comment:
>:attr:`play` is a :class:`~kivy.properties.BooleanProperty` and defaults to True.

Looks good  just need to change the docs below to say that it defaults to False

Reply:
Done